pepper_description/urdf/pepper1.0_generated_urdf/pepperGazebo.xacro

```diff
@@ -0,0 +1,779 @@
+      <frameName>BumperFL_frame</frameName>
+    </plugin>
+  </sensor>
+</gazebo>
+
+<gazebo reference="BumperFR_frame">
+  <mu1>0.5</mu1>
+  <mu2>0.5</mu2>
+  <selfCollide>false</selfCollide>
+  <turnGravityOff>false</turnGravityOff>
+  <fdir1>1 0 0</fdir1>
+  <maxVel>0.1</maxVel>
+  <minDepth>0.003</minDepth>
+  <sensor name="BumperFR" type="contact">
+    <always_on>1</always_on>
+    <update_rate>20</update_rate>
+    <visualize>false</visualize>
+    <contact>
+      <collision>BumperFR_frame_collision</collision>
+    </contact>
+    <plugin filename="libgazebo_ros_bumper.so" name="gazebo_pepper_BumperFR">
+      <robotNamespace>pepper_robot</robotNamespace>
+      <alwaysOn>true</alwaysOn>
+      <bumperTopicName>Bumper/FrontRight</bumperTopicName>
+      <frameName>BumperFR_frame</frameName>
+    </plugin>
+  </sensor>
+</gazebo>
+
+<!--
+    Sonars
+-->
+<gazebo reference="SonarBack_frame">
+  <sensor type="ray" name="SonarBack">
+    <pose>0 0 0 0 0 0</pose>
+    <update_rate>20</update_rate>
+    <visualize>false</visualize>
+    <ray>
+      <scan>
+        <horizontal>
+          <samples>5</samples>
+          <resolution>1</resolution>
+
+          <min_angle>-0.2617993877991494</min_angle>
+          <max_angle>0.2617993877991494</max_angle> 
+
+<!--
+          <min_angle>-0.10</min_angle>
+          <max_angle>0.10</max_angle> 
+-->
+        </horizontal>
+        <vertical>
+          <samples>5</samples>
+          <resolution>1</resolution>
+
+          <min_angle>-0.13</min_angle>
+          <max_angle>0.13</max_angle> 
+
+<!--
+          <min_angle>-0.2617993877991494</min_angle>
+          <max_angle>0.2617993877991494</max_angle> 
+-->
+        </vertical>
+      </scan>
+      <range>
+        <min>0.025</min>
+        <max>2.55</max>
+        <resolution>1</resolution>
+      </range>
+    </ray>
+    <plugin filename="libgazebo_ros_range.so" name="gazebo_ros_range">
+      <robotNamespace>pepper_robot</robotNamespace>
+      <gaussianNoise>0.05</gaussianNoise>
+      <alwaysOn>true</alwaysOn>
+      <updateRate>20</updateRate>
+      <topicName>sonar_back</topicName>
+      <frameName>SonarBack_frame</frameName>
+      <minRange>0.025</minRange>
+      <maxRange>2.55</maxRange>
+      <fov>0.5235987755982988</fov> 
+<!--      <fov>0.1</fov> -->
+      <radiation>ultrasound</radiation>
+    </plugin>
+  </sensor>
+</gazebo> 
+
+<gazebo reference="SonarFront_frame">
+  <sensor type="ray" name="SonarFront">
+    <pose>0 0 0 0 0 0</pose>
+    <update_rate>20</update_rate>
+    <visualize>false</visualize>
+    <ray>
+      <scan>
+        <horizontal>
+          <samples>5</samples>
+          <resolution>1</resolution>
+          <min_angle>-0.2617993877991494</min_angle>
+          <max_angle>0.2617993877991494</max_angle> 
+        </horizontal>
+        <vertical>
+          <samples>5</samples>
+          <resolution>1</resolution>
+          <min_angle>-0.13</min_angle>
+          <max_angle>0.13</max_angle>
```

Comment:
Are these sensor values accurate for pepper ? I remember tuning those by hand for Nao so I'm not sure that would apply for pepper